### ISSUE TITLE
Make add method in the class `Calculator`

### ISSUE THREAD

**Issue by mch5048** [kairproject/practice_collaboration#5]
1. implement an `add` method 
the method receives two real-valued inputs and outputs a sum of the two 

2. implement a test for `add` method
test will be done on the implemented script

Example) 
```
calc = Calculator()
# input can be any real number
x = 1.0
y = 1.0

assert calc.add(x, y) == 2.0
print ("Passed")

>>> Passed
```

**Comment by Curt-Park** [comment]
Why are the numbers integer values in the example test case?

**Comment by whikwon** [comment]
Comment updated `# input can be any real number`

**Comment by Curt-Park** [comment]
The example doesn't look like a test case.

**Comment by whikwon** [comment]
The above example represents code behavior not test case.

**Comment by whikwon** [comment]
Test case added

**Comment by mch5048** [comment]
> The example doesn't look like a test case.

modified the example code but not sure about how the test case may look like. please re-check

**Comment by Curt-Park** [comment]
Test case should be as follows:

x = 1.0  # real value
y = 1.0  # real value

~assert result == x+y~
assert  calc.add(x, y) == 2.0

**Comment by mch5048** [comment]
> Test case should be as follows:
> 
> x = 1.0 # real value
> y = 1.0 # real value
> 
> assert result == x+y
> assert calc.add(x, y) == 2.0

reflected comment

**Comment by Curt-Park** [comment]
@mch5048 not yet. please see the input values in the test case

**Comment by whikwon** [comment]
```
x = 1.0
y = 1.0
```
input values int to float updated

**Comment by Curt-Park** [comment]
Looks good to me. I will make a feature branch for you now.

**Comment by Curt-Park** [comment]
@mch5048 Please add this card on the project board (In progress condition)

**Comment by Curt-Park** [comment]
@mch5048 Please remove the last print line in your test case.

**Comment by Curt-Park** [comment]
Merged by 413a1ed47dfb5ff296b15b0a82b81b7e187099d5
